deployments/kustomize/image-builds/app.yaml

```diff
@@ -17,113 +6,49 @@ metadata:
     description: This is the image stream for the app run
   labels:
     role: app
-  name: gdx-agreements-tracker-app-run
+  name: app-run
 
 ---
-# BuildConfig app-build
+# BuildConfig app-run
 apiVersion: build.openshift.io/v1
 kind: BuildConfig
 metadata:
   annotations:
-    description: Compiles the React application for GDX agreements tracker.
+    description: Compiles the React application and uses Nginx (multi-stage build)for GDX agreements tracker.
   labels:
     role: app
-  name: gdx-agreements-tracker-app-build
+  name: app-run
 spec:
   failedBuildsHistoryLimit: 1
-  successfulBuildsHistoryLimit: 2
+  successfulBuildsHistoryLimit: 1
```

Comment:
Curious, why this was 2 before?

Reply:
i think all we need 1-2, just thought i would decrease, and see if we ever need the 2 in future.

Reply:
i think all we need 1-2, just thought i would decrease, and see if we ever need the 2 in future.